packages/typespec-ts/test/modularUnit/scenarios/clientContext/clientContext.md

```diff
@@ -167,6 +167,8 @@ export interface ServiceContext extends Client {}
 
 /** Optional parameters for the client. */
 export interface ServiceClientOptionalParams extends ClientOptions {
+  /** Need to be set as 'http://localhost:3000' in client. */
+  endpointParam?: string;
```

Comment:
Do we have a test case where endpoint has no default value? If no, please add one.